exporters/memory/include/opentelemetry/exporters/memory/in_memory_span_data.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "opentelemetry/sdk/trace/span_data.h"
+#include "opentelemetry/sdk/common/atomic_unique_ptr.h"
+#include "opentelemetry/sdk/common/circular_buffer.h"
+#include "opentelemetry/sdk/common/circular_buffer_range.h"
+#include "opentelemetry/sdk/trace/recordable.h"
+
+using opentelemetry::sdk::common::CircularBuffer;
+using opentelemetry::sdk::common::CircularBufferRange;
+using opentelemetry::sdk::common::AtomicUniquePtr;
+using opentelemetry::sdk::trace::Recordable;
```

Comment:
Could we move all unused headers into the `*.cc` file? Like `atomic_unique_ptr.h`, `circular_buffer_range.h` and `span_data.h`.

Also, let's avoid `using` directives in the header file. Everybody including the header file will get those, which can cause namespace pollution.

Reply:
Done